notebooks/MIRI_LRS_spectral_extraction/miri_lrs_advanced_extraction_part1.ipynb

```diff
@@ -0,0 +1,904 @@
+{
```

Comment:
<p>Another suggestion is to replace .format with f-string: </p><pre class="ql-syntax" spellcheck="false">print(f'Spectral extraction reference file used: {l3_spec.meta.ref_file.extract1d.name}')
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jdat_notebooks/pull/93/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>